Makefile

```diff
@@ -1,11 +1,11 @@
 
 install:
-	virtualenv .
+	python2.7 -m virtualenv .
```

Comment:
i think this needs a fairly recent version of python 2.7 .. ~2.7.11 ? wondering if tox is worth looking at.